src/algorithms/acquisition/adapters/beidou_b2ad_pcps_acquisition.cc

```diff
@@ -0,0 +1,310 @@
+/*!
+ * \file beidou_b2ad pcps_acquisition.cc
+ * \brief Adapts a PCPS acquisition block to an Acquisition Interface for
+ *  BEIDOU B2a signals
+ * \authors <ul>
```

Comment:
Mention to Google support should be added. Something like ` \note Code added as part of GSoC 2018 program`